ui/app/src/components/PreviewAudiencesPanel/PreviewAudiencesPanel.tsx

```diff
@@ -54,6 +55,7 @@ export function PreviewAudiencesPanel(props: PreviewAudiencesPanelProps) {
   const { fields } = props;
   const panelState = useSelection<GlobalState['preview']['audiencesPanel']>((state) => state.preview.audiencesPanel);
   const hasNoFields = !fields || Object.values(fields).length === 0;
+  const [timeZonesLookup, setTimeZonesLookup] = useSpreadState({});
```

Comment:
This being local state would get lost going in and out of the panel [as opposed to the rest of the state which is in redux or comes via props].

Where are the fields coming from?